src/http_server.h

```diff
@@ -35,10 +35,10 @@
 #include <queue>
 #include <string>
 #include <thread>
-#include <unordered_map>
 
 #include "common.h"
 #include "data_compressor.h"
+#include "python/tritonfrontend/_c/tritonfrontend.h"
```

Comment:
Why does http_server need to import python wrapper logic? Shouldn't it be the other way around?

Reply:
There were some helper functions in this file, that I was using `CreateWrapper()`, just removed the reference.